fews_3di/simulation.py

```diff
@@ -149,25 +149,41 @@ def run(self):
         self.simulation_id, self.simulation_url = self._create_simulation(model_id)
 
         laterals_csv = self.settings.base_dir / "input" / "lateral.csv"
-        laterals = utils.lateral_timeseries(laterals_csv, self.settings)
-        self._add_laterals(laterals)
+        if path.exists(laterals_csv):
+            laterals = utils.lateral_timeseries(laterals_csv, self.settings)
+            self._add_laterals(laterals)
+        else:
+            logger.info("No lateral timeseries found, skipping...")
 
         saved_state_id_file = self.settings.base_dir / SAVED_STATE_ID_FILENAME
         if self.settings.save_state:
             self._add_initial_state(saved_state_id_file)
             self.saved_state_id = self._prepare_initial_state()
+        else:
```

Comment:
Het is de vraag of deze info logging nodig is. Je zegt zelf expliciet in je settings dat je geen saved state wilt.

Of misschien de mededeling veranderen in zoiets als "Saved state not enabled in the configuration, skipping..."?